src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
+$day-size: 100px;
+$border-size: 1px;
+$day-gap: 1px;
+$day-background: #eee;
+$hover-background: #ffbfcb;
+$font-size: 30px;
+$acent-color: #000;
```

Comment:
There is a typo in the variable name `$acent-color`. It should be `$accent-color` for better clarity and consistency.